arkane/encorr/data.py

```diff
@@ -100,6 +100,12 @@
         'H': -0.5010030, 'N': -54.564343, 'O': -75.030991, 'C': -37.827717, 'P': -341.116432, 'S': -397.961110
     },
 
+    # g4 atom energies taken from [J. Chem. Phys. 126, 084108 (2007); doi: 10.1063/1.2436888]
+    'g4': {
+        'Br': -2573.58537, 'C': -37.83417, 'Cl': -460.01505, 'F': -99.70498,
```

Comment:
Could you add `S`, and `P`? (or, is there a reason not to add them?)
Could you also order the elements like the other model chemistries entries, so it's easier for users to compare what's missing?